bridge/gitea/export.go

```diff
@@ -0,0 +1,29 @@
+package gitea
+
+import (
+	"context"
+	"syscall"
+	"time"
+
+	"github.com/pkg/errors"
+
+	"github.com/MichaelMure/git-bug/bridge/core"
+	"github.com/MichaelMure/git-bug/cache"
+)
+
+var (
+	ErrMissingIdentityToken = errors.New("missing identity token")
+)
+
+// giteaExporter implement the Exporter interface
+type giteaExporter struct {
```

Comment:
this whole exporter could be removed (as it's not used), instead of returning those weird errors